services/autoscaling/src/simcore_service_autoscaling/modules/instrumentation.py

```diff
@@ -101,8 +120,11 @@ def setup(app: FastAPI) -> None:
     instrumentator = setup_rest_instrumentation(app)
 
     async def on_startup() -> None:
+        metrics_subsystem = (
+            "dynamic" if app_settings.AUTOSCALING_NODES_MONITORING else "computational"
+        )
         app.state.instrumentation = AutoscalingInstrumentation(
-            registry=instrumentator.registry
+            registry=instrumentator.registry, subsystem=metrics_subsystem
```

Comment:
Would it make sense to do this also for the "default" metrics collected? This can be done like [this](https://github.com/trallnag/prometheus-fastapi-instrumentator?tab=readme-ov-file#specify-namespace-and-subsystem). Of course then I would need to adapt the Grafana dashboards

Reply:
that is why I did not touch them.